alerting/src/test/kotlin/com/amazon/opendistroforelasticsearch/alerting/TestHelpers.kt

```diff
@@ -220,13 +221,27 @@ fun Monitor.toJsonString(): String {
 }
 
 fun randomUser(): User {
-    return User("joe", listOf("ops", "backup"), listOf("all_access"), listOf("test_attr=test"))
+    val ranStrGen = RandomStringGenerator.Builder().build()
+    val randomUser = ranStrGen.generate(5)
+    val bckEndRole1 = ranStrGen.generate(10)
+    val bckEndRole2 = ranStrGen.generate(10)
```

Comment:
There are already provided helper methods for generating random values, e.g. `ESRestTestCase.randomAlphaOfLength(10)`